src/api/app/models/staging/staged_requests.rb

```diff
@@ -84,6 +85,34 @@ def requests
     staging_workflow.unassigned_requests.where(number: request_numbers)
   end
 
+  def link_package(bs_request_action)
+    request = bs_request_action.bs_request
+
+    source_package = Package.get_by_project_and_name!(bs_request_action.source_project,
+                                                      bs_request_action.source_package)
+
+    # get the base revision from the source package
+    baserev = source_package.backend_package.srcmd5
```

Comment:
This is not the submitted revision but HEAD

Reply:
fixed, could you look into it again @coolo ?